src/Categories/Diagram/Coequalizer/Properties.agda

```diff
@@ -33,16 +33,13 @@ module _ (coe : Coequalizer f g) where
     equalizer = Coequalizer⇒coEqualizer coe
 
   open Equalizer equalizer
-    using (unique′; equality-∘; unique-diagram)
+    using (unique′; unique-diagram)
     renaming ( id-equalize      to id-coequalize
              ; equalize-resp-≈  to coequalize-resp-≈
              ; equalize-resp-≈′ to coequalize-resp-≈′
              )
     public
 
-  Coequalizer⇒Epi : Epi arr
```

Comment:
This proof was moved to `Categories.Diagram.Coequalizer`, as it's a one liner, and proving it by duality here makes the module graph a bit messy.